<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BarCodeDlg</class>
 <widget class="QDialog" name="BarCodeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>条形码定位</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>条形码定位</string>
     </property>
     <layout class="QGridLayout" name="groupLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="topLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>条形码类型：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeCmb">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <item>
           <property name="text">
            <string>试卷类型</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>准考证号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>试室号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>缺考</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="midLayout">
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>框选条形码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="text">
           <string>确认条形码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="bottomLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="deleteBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>55</height>
           </size>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="codebarList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>45</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>typeCmb</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>confirmBtn</tabstop>
  <tabstop>codebarList</tabstop>
  <tabstop>deleteBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
